src/query/v2/trigger_context.cpp

```diff
@@ -265,7 +265,7 @@ void TriggerContext::AdaptForAccessor(DbAccessor *accessor) {
     // adapt created_vertices_
     auto it = created_vertices_.begin();
     for (auto &created_vertex : created_vertices_) {
-      if (auto maybe_vertex = accessor->FindVertex(created_vertex.object.Gid(), storage::v3::View::OLD); maybe_vertex) {
+      if (auto maybe_vertex = accessor->FindVertex(kFakeVertexGid); maybe_vertex) {
         *it = detail::CreatedObject{*maybe_vertex};
```

Comment:
The triggers will be completely broken, so the only goal here is to minize the time you spend on fixing this.